<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPlotHistory</class>
 <widget class="QWidget" name="FormPlotHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotHistory</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,9">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="9,9">
     <item>
      <layout class="QVBoxLayout" name="vLay14">
       <item>
        <layout class="QHBoxLayout" name="hLay14">
         <item>
          <widget class="QLabel" name="label14">
           <property name="text">
            <string>curve 14bit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnPrev14">
           <property name="text">
            <string>prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnNext14">
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLay14Status">
         <item>
          <widget class="QLabel" name="labelStatus14">
           <property name="text">
            <string>status</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLay14StatusGo">
           <item>
            <widget class="QLabel" name="label14Go">
             <property name="text">
              <string>go:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit14Go"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vLay24">
       <item>
        <layout class="QHBoxLayout" name="hLay24">
         <item>
          <widget class="QLabel" name="label24">
           <property name="text">
            <string>curve 24bit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnPrev24">
           <property name="text">
            <string>prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnNext24">
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLay24Status">
         <item>
          <widget class="QLabel" name="labelStatus24">
           <property name="text">
            <string>status</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLay24StatusGo">
           <item>
            <widget class="QLabel" name="label24Go">
             <property name="text">
              <string>go:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit24Go"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hLayOpt">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonDumpPlot">
       <property name="text">
        <string>dump plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAll">
       <property name="text">
        <string>all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonDumpData">
       <property name="text">
        <string>dump data</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vLayFitting">
       <item>
        <layout class="QHBoxLayout" name="hLayFittingSin">
         <item>
          <widget class="QToolButton" name="tBtnFittingSin">
           <property name="text">
            <string>FittingSin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_y0">
           <property name="text">
            <string>y0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_y0"/>
         </item>
         <item>
          <widget class="QLabel" name="label_xc">
           <property name="text">
            <string>xc</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_xc"/>
         </item>
         <item>
          <widget class="QLabel" name="label_w">
           <property name="text">
            <string>w</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_w"/>
         </item>
         <item>
          <widget class="QLabel" name="label_A">
           <property name="text">
            <string>A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_A"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxFittingSinConversion">
           <property name="text">
            <string>conversion</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayFittingKB">
         <item>
          <widget class="QToolButton" name="tBtnFittingKB">
           <property name="text">
            <string>FittingKB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelK">
           <property name="text">
            <string>k</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditK"/>
         </item>
         <item>
          <widget class="QLabel" name="labelB">
           <property name="text">
            <string>b</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditB"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxConversion">
           <property name="text">
            <string>conversion</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonSplit">
       <property name="text">
        <string>Split</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonMix">
       <property name="text">
        <string>Mix</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
